src/components/Error/index.js

```diff
@@ -1,21 +1,38 @@
-import { h } from 'preact'
+import { h, Component } from 'preact'
 import {errors} from '../strings/errors'
 import style from './style.css'
+import { sendScreen } from '../../Tracker'
 
-const Error = ({error}) => {
-  const errorText = errors[error.name]
-  const errorType = error.type === 'error' ? 'error' : 'warning';
-  return (
-    <div className={style[`container-${errorType}`]}>
-      <div className={style.title}>
-        <span className={style[`title-icon-${errorType}`]}/>
-        <span className={style['title-text']}>{errorText.message}</span>
+class Error extends Component {
+  componentDidMount() {
+    const errorName = this.props.error.name.toLowerCase()
+    if (this.errorType() === 'error' && this.isSmsError()) {
+      sendScreen([errorName])
+    }
+  }
+
+  isSmsError = () => {
+    const errorName = this.props.error.name
+    return errorName === 'SMS_FAILED' || errorName === 'SMS_OVERUSE'
```

Comment:
why is this necessary?

Reply:
We only want to send the screen to woopra if it's an sms related error

Reply:
what are the other cases?
And if that is the cases then it's better to have a component called SMSError screen that sends that error, rather than having this complexity here.

Reply:
These are the other use cases:
```'INVALID_CAPTURE'
  'INVALID_TYPE'
  'UNSUPPORTED_FILE'
  'INVALID_SIZE'
  'NO_FACE_ERROR'
  'MULTIPLE_FACES_ERROR'
  'SERVER_ERROR'
  'GLARE_DETECTED'```
All of these errors are tracked on the uploader screen, the one that is currently displaying the wrong message on Woopra (it always tracks an error screen for uploader. I could track all the errors in the Error component and this would also fix the issue for the uploader component.